eng/pipelines/templates/jobs/vmr-build.yml

```diff
@@ -339,6 +351,12 @@ jobs:
           ./prep-source-build.sh $customPrepArgs
         displayName: Prep the Build
 
+    - ${{ if ne(parameters.runOnline, 'True') }}:
+      - script: |
+          set -ex
+          sudo ip link set eth0 down
```

Comment:
`ip` is not installed. I can either adjust the Dockerfiles or try to install it here, but that would llikely involve trying to work around different distros' package managers.

Reply:
@mthalman or @lbussell - Do you have thoughts on the above?

Also related, I tested the above command out by installing `iproute2` in a local docker container using the ubuntu 24.04 buildtools prereqs image. I found that I needed privileged access to run `ip link set eth0 down`. Given that the container is not manually configured, I worry about the fact that I may not be able to run the command in the pipelines even after installing `iproute2`. I don't think that the use of `sudo` fixes this issue.

Reply:
It looks like you need to run the container with privileged access to be able to run that command.

This can be configured for the container job:

```yaml
container:
  image: <your-container-image>
  options: --privileged
```

> I can either adjust the Dockerfiles or try to install it here in the pipeline.

It should be added to the Dockerfiles.

Reply:
> It should be added to the Dockerfiles.

For testing purposes, you can install it dynamically in the container to verify all that you'll need to have installed before updating the Dockerfiles.